src/web.rs

```diff
@@ -93,7 +93,7 @@ impl WebInterface {
     fn expose_openapi_json(&mut self) {
         self.server
             .at("/v1/openapi.json")
-            .get(|req: Request<()>| async move {
+            .get(|_req| async move {
```

Comment:
`||` instead of `|_req|`?

Reply:
The `.get` method expects a closure with one parameter, so that won't work.
We could go with `|_|` instead of `|_req|` but I found the latter to be a bit more descriptive in terms of *what* parameter we are ignoring. 

```
   Compiling tacd v0.2.0 (/home/leonard/Documents/Projects/tacd)
error[E0593]: closure is expected to take 1 argument, but it takes 0 arguments
   --> src/web.rs:96:14
    |
96  |             .get(|| async move {
    |              ^^^ -- takes 0 arguments
    |              |
    |              expected closure that takes 1 argument
    |
    = note: required because of the requirements on the impl of `Endpoint<()>` for `[closure@src/web.rs:96:18: 96:20]`
```